### DISCUSSION TITLE
Dashboard sidebar not showing headings

### DISCUSSION THREAD

**Discussion by JosiahParry** [quarto-dev/quarto-cli#7544] [Q&A] [answered]
### Description

When using the `{.sidebar}` class with a quarto dashboard the section headings are removed. How can I get them to persist? 

<img width="793" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/13534508/3afd91d0-76d8-4904-aeff-6a0c584ea034">

````
---
title: "Help me! "
author: "Josiah Parry"
format: 
  dashboard:
    orientation: columns
    scrolling: false
    theme: zephyr
    
---

# {#instructions .sidebar width=25%}

### Global Moran's I

Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt 


```r
library(sfdep)
```

Faucibus purus in massa tempor nec. Placerat vestibulum lectus mauris ultrices eros in cursus.

### Doing some other stuff 

```r
nb <- st_contiguity(x)
wt <- st_weights(nb)
```
 
Amet mauris commodo quis imperdiet massa tincidunt nunc pulvinar. Fermentum dui faucibus in ornare quam viverra orci. Nulla at volutpat diam ut venenatis tellus in metus.

# 

  
## {#source width="37.5%"}

![](https://placehold.co/50)


## {width="37.5%"}

###

section

![](https://placehold.co/100)


###

section 

![](https://placehold.co/100)
````

**Comment by jjallaire** [comment]
I think headings are supposed to work as normal in sidebars. @dragonstyle  ?

**Reply by dragonstyle** [reply]
Yes, it shouldn't be processing the subheadings into layouts/cards. I will have a closer look!

**Reply by JosiahParry** [reply]
Thanks both! Any chance you were able to identify what I might be doing wrong here?

**Reply by dragonstyle** [reply, marked as answer]
You were doing nothing wrong - this was a Quarto bug. I have fixed that issue so your example above works (for me at least) with no modification in the latest pre-release build. 

I meant to follow up here once I fixed the issue - apologies for not doing so sooner!

**Reply by JosiahParry** [reply]
No worries! Thank you for letting me know. More often than not it's me doing something wrong :)
